scripts/dep-guard.sh

```
@@ -358,15 +297,15 @@ cmd_check_ripple_keyword_v1() {
   fi
 
   if [[ -z "$src_title" ]]; then
-    echo "⚠️  Warning: could not extract title for ${issue_id} — ripple check skipped" >&2
+    echo "âš ï¸  Warning: could not extract title for ${issue_id} â€” ripple check skipped" >&2
```

Shellcheck (0.11.0)

[warning] This is a unicode quote. Delete and retype it (or ignore/singlequote for literal).

(SC1111)